src/documentation/pages/components/forms/search-input/index.vue

```diff
@@ -0,0 +1,35 @@
+<template lang="pug">
+  div
+    <TitleBar title="Search input" description="Search inputs are used for inputting words or phrases and locating relevant content." :tabs="pageTabs" />
+    .chi-grid__container.-pt--3
+      .chi-tabs-panel.-active#examples
+        <Examples />
+      .chi-tabs-panel#properties
+        <Properties />
+      .chi-tabs-panel#accessibility
+        <Accessibility />
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+import Examples from './examples/index.vue';
+import PageContentTabs from '../../../../components/PageContentTabs.vue';
+import Properties from './_properties.vue';
+import Accessibility from './_accessibility.vue';
+import { standardComponentPageTabs } from '../../../../constants/constants';
+
```

Comment:
Please add Vue.config.ignoredElements to root-most index.vue to ignore chi-search-input, chi-brand etc